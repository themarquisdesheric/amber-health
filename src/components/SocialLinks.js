import React from 'react';

import instagram from '../img/social/instagram.svg';
import twitter from '../img/social/twitter.svg';

const MakeSocialLink = ({ title, href, src }) => (
  <a title={title} href={href} target="_blank" rel="noopener noreferrer">
    <img
      src={src}
      alt={title}
    />
  </a>
);

const SocialLinks = () => (
  <span className="social">
    <MakeSocialLink title="instagram" href="https://instagram.com" src={instagram} />
    <MakeSocialLink title="twitter" href="https://twitter.com" src={twitter} />
  </span>
);

export default SocialLinks;
