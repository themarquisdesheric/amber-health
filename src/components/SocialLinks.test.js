import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SocialLinks from './SocialLinks';

test('SocialLinks', () => {
  const wrapper = shallow(<SocialLinks />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
