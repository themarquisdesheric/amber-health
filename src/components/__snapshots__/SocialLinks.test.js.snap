// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialLinks 1`] = `
<span
  className="social absolute block w-full text-center"
>
  <MakeSocialLink
    href="https://instagram.com"
    src="test-file-stub"
    title="instagram"
  />
  <MakeSocialLink
    href="https://twitter.com"
    src="test-file-stub"
    title="twitter"
  />
</span>
`;
